### PR TITLE
Update aip-36.md

### PR DESCRIPTION
**Pull request by igor-aptos**
Update description from recent discussions and changes from the PR

### CODE REVIEW

File: aips/aip-36.md [hunk 1]
```diff
@@ -12,15 +12,21 @@ created: 06/01/2023
...
+This AIP thereby creates a new type of identifier called UUID (Universally Unique Identifier). This is a 256-bit identifier that is universally unique amongst all the identifiers generated by all the accounts for all purposes (and are different from other methods generating identifiers that use domain separation from `Scheme` in `authenticator.rs`). We propose adding two native functions `create_unique_address` and `get_txn_hash` in Aptos framework. Every time a MOVE code calls `create_unique_address`, the function outputs a universally unique 256-bit value (of type `address`). The function creates a unique address by using the hash of the current transaction. 
+As exposing the transaction hash might have more applications in the future, we also propose to add the `get_txn_hash` function which returns the hash of the current transaction.
+
+We additionally create `object::create_object` and `token::create_token` functions, utilizing newutility functions, to provide more efficient untracked token minting. Those deprecate `object::create_object_from_account`, `object::create_object_from_object` and `token::create_from_account` functions.
```
Comment (movekevin) [inline aips/aip-36.md:27]: newutility -> new utility

**Review by movekevin — APPROVED** [review]

File: aips/aip-36.md [hunk 2]
```diff
@@ -12,15 +12,21 @@ created: 06/01/2023
...
+This AIP proposes to add two new native functions `create_unique_address` and `get_txn_hash` in Aptos framework. The `create_unique_address` generates and outputs a unique 256-bit identifier (of type address) for each function call. The `get_txn_hash` function outputs the hash of the current transaction. The `create_unique_address` function calls can run efficiently and in parallel. In other words, when two transactions run `create_unique_address` method, they can be executed in parallel without any conflicts. Initially, we will use these unique identifiers internally as addresses for newly created Move Objects, specifically in Token V2 minting.
+We also propose to add a wrapper function called `create_uuid` which wraps the unique address generated by the `create_unique_address` in the non-copy-able UUID struct type, guaranteeing that every two instances of UUID value are different.
```
Comment (gedigi) [inline aips/aip-36.md:16]: is it OK to call this struct `UUID` when it doesn't meet the `UUID` spec (https://datatracker.ietf.org/doc/html/rfc4122)?

File: aips/aip-36.md (hunk 1 above)
Comment (igor-aptos) [inline aips/aip-36.md:27]:
```suggestion
We additionally create `object::create_object` and `token::create_token` functions, utilizing new utility functions, to provide more efficient untracked token minting. Those deprecate `object::create_object_from_account`, `object::create_object_from_object` and `token::create_from_account` functions.
```

**Review by thepomeranian — APPROVED** [review]

**Review by thepomeranian — APPROVED** [review]

**Review by thepomeranian — APPROVED** [review]

File: aips/aip-36.md (hunk 2 above)
Comment (igor-aptos) [inline aips/aip-36.md:16]:
```suggestion
We also propose to add a wrapper function called `generate_auid` which wraps the unique address generated by the `generate_unique_address_internal` in the non-copy-able AUID struct type, guaranteeing that every two instances of AUID value are different.
```

**Pull request merged**